site/cypress/e2e/grading_stats.spec.js

```diff
@@ -0,0 +1,111 @@
+import { skipOn } from '@cypress/skip-test';
+
+skipOn(Cypress.env('run_area') === 'CI', () => {
+    describe('Grading stats as an instructor', () => {
+        beforeEach(() => {
+            cy.visit(['sample']);
+            cy.login('instructor');
+        });
+        afterEach(() => {
+            cy.logout();
+        });
+        it('should be accurate for teams.', () => {
+            cy.get('a[href*="/sample/gradeable/grading_team_homework/grading/status"]').click();
```

Comment:
This line fails when testing on my machine (0 matched elements)
`cy.get('a[href*="/sample/gradeable/grading_team_homework/grading/status"]').click();`